### ISSUE TITLE
problem gamepad need for speed rivals 

### ISSUE THREAD

**Issue by ilyessschaf** [ilyessschaf/ilyess#1]
Hey, i bought Giotech VX-2 Wired controller few days ago, it's for PS3(/PC). I have followed the instructions step by step on internet or youtube. All works fine in the X360ce application recognizes everything and i press save a. In Need For Speed Rivals i had to press A (x) to continue but it didn't work. I pressed all the buttons and R1 worked (as A/X). And B (circle) was pad down and Y (triangle) pad up and few other. ANY button/stick didn't work right.nd go in the game. The buttons are really messed up

**Comment by ilyessschaf** [comment]
Hey, i bought Giotech VX-2 Wired controller few days ago, it's for PS3(/PC). I have followed the instructions step by step on internet or youtube. All works fine in the X360ce application recognizes everything and i press save a. In Need For Speed Rivals i had to press A (x) to continue but it didn't work. I pressed all the buttons and R1 worked (as A/X). And B (circle) was pad down and Y (triangle) pad up and few other. ANY button/stick didn't work right.nd go in the game. The buttons are really messed up
